resources/corretto_install.rb

```diff
@@ -0,0 +1,87 @@
+resource_name :corretto_install
+include Java::Cookbook::CorrettoHelpers
+default_action :install
+
+property :version, String, name_property: true
```

Comment:
descriptions?

Reply:
oh we can do that now!

Reply:
As long as you dep on Chef 14+

Reply:
Yeah I'm aiming for this to be out when 16 is out, and to siplify things I've depd on 15 anyway